src/core/column-type/number.column.js

```diff
@@ -19,6 +19,7 @@ TemplatePath.register('number-cell-edit', (template, column) => {
 class NumberColumnModel extends DataColumnModel{
 	constructor(){
 		super('number');
+		this.editorSubmitShortcut = 'tab';
```

Comment:
I think we will put to the edit model, and remove from column. It doesn't look natural here. Can you put it to
model.edit().commitShortcuts = {
'$default': 'tab|enter',
'date': 'enter',
...
};

something like that? or maybe in navigation model?